tests/kern/pathwalk.c

```diff
@@ -149,8 +150,8 @@ main (int argc, char *argv[])
             err_exit ("gettimeofday");
         timersub (&t2, &t1, &elapsed);
         if (!qflag)
-            msg ("Created %d objects in %lu.%.3lus", length * files + 1,
-                 elapsed.tv_sec, elapsed.tv_usec / 1000);
+            msg ("Created %d objects in %"PRIdMAX".%.3"PRIdMAX"s", length * files + 1,
+                 (intmax_t)elapsed.tv_sec, (intmax_t)elapsed.tv_usec / 1000);
```

Comment:
Thanks!

The format specifier macros are kind of clunky to use.  Are they helpful here?   In other projects we've just used `%jd` for an `intmax_t` argument.   In a quick spot check of linux, freebsd, and macos, I'm seeing `j` in all the printf(3) man pages.

Any reason not to go that way?

Reply:
I just thought this was guaranteed to be portable.  But as you say that might not be an issue for `intmax_t` . I can change it if you want.

Reply:
If you don't mind - please.

Reply:
I've pushed a new version using plain `%jd`.